test/extended/testdata/builds/docker-add/docker-add-env/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM centos
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
registry.redhat.io

Reply:
or did you use registry.access.redhat.com because it doesn't need pull creds?  (i think our CI clusters have pull creds for registry.redhat.io and if so, i think that would be preferred)

Reply:
that was my  thinking (wasn't sure if it worked for imagestreams only or all image pulls) but let's try it and see

fwiw the test in question passed in e2e-gcp-ovn-builds with the registry.access image:

`:  [sig-builds][Feature:Builds][Slow] build can have Dockerfile input  being created from new-build testing build image with dockerfile  contains a file path uses a variable in its name  [apigroup:build.openshift.io] | 43s`

Reply:
it passed again in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/27718/pull-ci-openshift-origin-master-e2e-gcp-ovn-builds/1624089082722258944 @bparees 

we should be good to go